histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import ctypes
+
+from histomicstk.segmentation.label.isbf import isbf
```

Comment:
Change this to `from .isbf import isbf`. That should resolve the TravisCI error. In general, if your are import from a file in the same folder use relative import like this. Otherwise, use absolute import.